### ISSUE TITLE
N-Rage Controller Plugin GUI

### ISSUE THREAD

**Issue by CCrashCup** [project64/project64#2033]
This is just a suggestion.
Change the "Plugged" label on the check box to say "Connected" instead.
Since there are wireless controllers, it can be confusing. And people don't always understand even when their controller is wired.

## Steps to reproduce the problem

  1.
  1.
  1.
  1.
  1.

## Specifications

  - Windows version:
  - Project64 version:
  - Plugins used:

**Comment by IronYoshi64** [comment]
Yeah I agree this would be a Very nice change, plus it would make more sense.

**Comment by vgturtle127** [comment]
I'll take a look later today and see if I can find the strings related and update them

**Comment by Squall-Leonhart** [comment]
leave em alone, editing the resources breaks compilation.
